src/features/commands.ts

```diff
@@ -769,6 +769,39 @@ https://exploringjs.com/es6/ch_variables.html#_pitfall-const-does-not-make-the-v
       });
     },
   },
+  {
+    words: [!remote, !remotework],
```

Comment:
```suggestion
    words: ["!remote", "!remotework"],
```

Reply:
Sorry about that. It must've somehow changed from a string to vars when I tried to copy and paste this blurb I wrote out elsewhere. Fixed!